app/controllers/application_controller.rb

```diff
@@ -1,10 +1,29 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
-  helper_method :cart
+  helper_method :cart, :current_user, :user
+  # refactor later to keep either `current_user` OR `user`
 
   def cart
     @cart ||= Cart.new(session[:cart] ||= Hash.new(0))
   end
 
+  def current_user
+    @current_user ||= User.find(session[:user_id]) if session[:user_id]
+  end
+
+  def user
```

Comment:
a bit confused by this - I thought we were keeping `current_user`? 
I'd like to delete the rest we won't be using to avoid confusion before merging to master (so by the next pull request)

Reply:
When this gets merged I have a PR ready.